internal/cluster/connection.go

```diff
@@ -297,7 +297,7 @@ func positiveDurationOrMax(duration time.Duration) time.Duration {
 	if duration > 0 {
 		return duration
 	}
-	return time.Duration(math.MaxInt64)
+	return time.Duration(validationutil.MaxInt)
```

Comment:
`time.Duration` is already `int64`, so `math.MaxInt64` should be OK. Does it cause any problems?

Reply:
Checked tests, looks good. Thanks